lib/DDG/Spice/GetEvents/Nearby.pm

```diff
@@ -1,15 +1,15 @@
-package DDG::Spice::GetEvents;
+package DDG::Spice::GetEvents::Nearby;
```

Comment:
@Uriger sorry, what I meant was to just use the city API endpoint, and when the query is "events near me", instead of taking the user's lat/ln, we take `$loc->city` and pass that to the API which simplifies everything.

>What is $result? What should be it's value?
>What is is_local? Is that just a boolean variable that tells in which case are we?

These were just meant to be placeholders to show how the logic would work. Meaning you'd need to determine if we're dealing with a "local events" query or not. If so, we return whatever result is necessary (i.e. the users city) and set the caching off. Otherwise, we return the city given in the query and leave caching on.